File: Php/app/Urls/PlayerDetailsUrlModel.php
namespace app\Urls{

	use app\RouteFormats;
	use app\Urls\BaseClasses\PlayerUrlModel;
	use entities\Player;
	use entities\Homegame;

	class PlayerDetailsUrlModel extends PlayerUrlModel{

		public function __construct(Homegame $homegame, Player $player){
			parent::__construct(RouteFormats::playerDetails, $homegame, $player);
		}

	}

}

File: Web/Models/Url/PlayerDetailsUrlModel.cs
using Core.Classes;
using Web.Routing;
using app;

namespace Web.Models.Url{

	class PlayerDetailsUrlModel : PlayerUrlModel{

		public PlayerDetailsUrlModel(Homegame homegame, Player player): base(RouteFormats.PlayerDetails, homegame, player)
        {
		}

	}

}